tests/util/test_math.py

```diff
@@ -0,0 +1,49 @@
+import scipy
+import scipy.integrate
+from scipy.stats import norm
+from numpy import log, sqrt, exp, inf
+
+from util import test_statistic_threshold as tst
+
+
+def kullback_leibler(post_sample, post_pdf, comp_pdf):
+    """Estimate KL-divergence of sample (a collection of values) w.r.t. known pdf,
+    comp_pdf, which returns the density when passed a sample. Return value is
+    (estimated_kl, variance) post_sample is an approximate sample from the
```

Comment:
Can you clarify what variance this is? Estimated sampling variance of the KL divergence?